src/core/etl/src/Flow/ETL/Function/ScalarFunctionChain.php

```diff
@@ -150,9 +151,9 @@ public function greaterThanEqual(ScalarFunction $ref) : self
         return new GreaterThanEqual($this, $ref);
     }
 
-    public function hash(string $algorithm = 'xxh128', bool $binary = false, array $options = []) : self
+    public function hash(?Algorithm $algorithm = null) : self
```

Comment:
Instead of making it nullable I would use `hash(Algorithm $algorithm = new NativePHPHash()) : self`

Reply:
Fixed, sorry but I forgot it is possible since 8.1 :D

Reply:
no worries, took me a while to get used to it